src/handler/handleRoleReaction.ts

```diff
@@ -1,15 +1,24 @@
 import type { RoleReaction, Services } from "../index.types";
 
 export const handleRoleReaction = (services: Services) => async ({
+  reactorType,
   type,
   reaction,
   member,
 }: RoleReaction): Promise<void> => {
   try {
-    if (type === "add") {
-      await services.roleReactor.applyRole(reaction, member);
+    if (reactorType === "group") {
```

Comment:
To me, this looks a bit repetitive/duplicated. Maybe this can be simplified to something like:
```ts
const reactor = reactorType === "group" ? services.roleReactor : services.colorReactor;
if (type === "add") {
  await reactor.applyRole(reaction, member);
} else {
  // etc...
}
```